### DISCUSSION TITLE
Not able to find the Losshitory plots using saveplot

### DISCUSSION THREAD

**Discussion by anshumansinha16** [lululxvi/deepxde#966] [Q&A]
I am trying to plot the losshistory (i.e MSE loss in Train, Test data vs epochs) in order to analyse my simulations. As I am performing a parametric study by varying many parameters, I am trying to save all the loss history while my simulation is running and then later on plot them one after the other at the end of the simulation!

Currently, the code is plotting them after every loop of each parameter (i.e if I have 10*5 parameters , then I am getting 50 plots periodically after each simulation (end of epochs)! This is becoming very crumble-some for me to wait and attend all the 50 plots one after the other during the run-time! That's why I want to look at these after all my simulations have finished! For that I enabled the issave=True. But was not bale to locate the plots!

1. I am not able to know where are my dde.saveplot files are being saved?
2. And what will be the name of these files? (Name or extension; such that I can find them easily in the folder)?
3. While keeping the isplot=True; Can there be anyway in which my terminal does not pauses to show the plots? That is, it can plot the error figures and also run in the background, hence all the plots are plotted without me going and closing each one of them one by one and then the program restarting?

**Error:** On executing this command; the following error shows up on the terminal Saving loss history to /Users/anshumansinha/Desktop/Project/file/loss.dat .... Error: The network has multiple inputs, and saving such result han't been implemented.

**Code:**

    model = dde.Model(data, net)
    model.compile("adam", lr=lr, metrics=[mean_squared_error])
    checker = dde.callbacks.ModelCheckpoint(
        "/Users/anshumansinha/Desktop/Project/model/"+save_str+"model.ckpt", save_better_only=False, period=100
    )

    losshistory, train_state = model.train(epochs=epochs, callbacks=[checker]) #Training Model batch_size = 10000

    dde.saveplot(losshistory, train_state, issave=True, isplot=True)

**Comment by tsarikahin** [comment]
Because it is not implemented for higher dimensions, it is available only for 1D problems

**Reply by anshumansinha16** [reply]
Thanks for the reply! Do you have any idea how to go about and solve this problem? Because I am able to get plots when I turn `issave = False` and leave `isplot = True`! 

I want to save all the plots without having to pause in the middle of the simulation; or else a plot appears in the middle of the simulation and I need to close it to move forward with the simulation and I need to do this for like 50 odd times?

**Reply by tsarikahin** [reply]
I think you can use `model.predict` for that. After training, create new points using `geom` and make predictions so that you can use those predictions for plotting. It should be straightforward.
